<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolTable</class>
 <widget class="QDialog" name="ToolTable">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Define Tool Table</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
selection-background-color: lightblue;
background-color:#cdcdcd;
}

QLabel {
font: 75 16pt &quot;MathJax_SansSerif&quot;;
color: fcfcfc;
}

QLabel#logoL {
background-color: #fcfcfc;
}

QPushButton {
font: 75 11pt &quot;MathJax_SansSerif&quot;;
  border: 1px solid #ddd;
  border-radius: 3px;
  background-color: #fcfcfc;
  padding: 4px;
  color: #e5aa3d;
  line-height: 1.5em;
}
QPushButton:hover {
  color: #428bca;
  background-color: #ddd;
}

QPushButton:disabled,
QPushButton:hover:disabled,
QPushButton.disabled,
QPushButton.disabled:hover {
  background: #ededed;
  color: #888 !important;
  border-color: #ddd;
}

QLineEdit {
font: 75 11pt &quot;MathJax_SansSerif&quot;;
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
color: black;
background-color: white;
selection-background-color: lightblue;
}

QFrame {
border-style: none;
}

QComboBox {
font: 75 11pt &quot;MathJax_SansSerif&quot;;
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
background-color: 'white';
selection-background-color: lightblue;
}

QTableWidget {
font: 75 11pt &quot;MathJax_SansSerif&quot;;
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
selection-background-color: lightblue;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame_17">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="logoL">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#ffffff;&quot;&gt;Learn more at buildbotics.com!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>logo side by side.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="ttNameLabel">
        <property name="text">
         <string>Tool Table Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameLE">
        <property name="toolTip">
         <string>Provide a descriptive name for this tool table.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addBtn">
           <property name="toolTip">
            <string>Add a new tool to the list.</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="list-add">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeBtn">
           <property name="toolTip">
            <string>Delete the selected tool from the list.</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="list-remove">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editBtn">
           <property name="toolTip">
            <string>Edit the selected tool.</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="accessories-text-editor">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="upBtn">
           <property name="toolTip">
            <string>Move the selected tool up
by one position in the list.</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="go-up">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="downBtn">
           <property name="toolTip">
            <string>Move the selected tool down by
one position in the tool list.</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="go-down">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Number</string>
         </property>
         <property name="toolTip">
          <string>Displays the tool number.</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
         <property name="toolTip">
          <string>Displays the tool type.</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
         <property name="toolTip">
          <string>Displays the name of this tool.</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="toolTip">
      <string>&quot;Apply&quot; saves changes but does not close the dialog.
&quot;Cancel&quot; discards changes and closes the dialog.
&quot;OK&quot; saves the changes and closes the Tool Table dialog.</string>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
